packages/examples/src/requestDepositAddress.ts

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env node --import=tsx --trace-uncaught --no-warnings
+
+import { SwapSDK, Chains, Assets } from '@/sdk/swap/index.js';
+import 'dotenv/config';
+
+const swapSDK = new SwapSDK({
+  network: 'perseverance',
+  enabledFeatures: { dca: true },
+  broker: { url: 'http://10.5.2.143:8080/ ' },
+});
+
+const { quotes } = await swapSDK.getQuoteV2({
```

Comment:
not related to this pr but i had it on my stash and its convenient to have it in the repo to for debugging